float64 raise
float64 pitch
float64 omega